tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
     "lib": [
-      "esnext"
+      "esnext",
+      "dom"
```

Comment:
Why?

Reply:
There was an issue when the dom was not added. Now it fixes by itself.